remote_config/src/desktop/remote_config_response.cc

```diff
@@ -90,6 +90,9 @@ Variant RemoteConfigResponse::GetEntries() { return entries_; }
 // Mark the response completed for both header and body.
 void RemoteConfigResponse::MarkCompleted() {
   ResponseJson::MarkCompleted();
+  if (strlen(GetBody()) == 0) {
```

Comment:
Since no other lines of code use GetBody() in this function, but instead call into flatbuffers, I think it's worth writting a comment above this line to describe why this is a shortcut out of the function.

Reply:
It's also faster not to strlen() the whole body in case it's quite large. Just check if GetBody()[0] == '\0'.

Reply:
DONE